src/pspm_import.m

```diff
@@ -32,7 +32,9 @@
 %              Can be one of the following units:'mm', 'cm', 'm','inches'.
 %           - .denoise: for marker channels in CED spike format (recorded 
 %              as 'level'), filters out markers duration longer than the 
-%              value given here (in ms) 
+%              value given here (in ms)
+%           - .delimiter: for delimiter separated values, value used as delimiter for file read
+% 

```

Comment:
If in `pspm_import` you put the `delimiter` in the `import` structure, why in `pspm_get_txt` you separate it from the structure ? (comment related to the comment in `pspm_get_txt`)

Reply:
:+1: